codec-http2/src/main/java/io/netty/handler/codec/http2/HttpUtil.java

```diff
@@ -324,6 +326,32 @@ public boolean visit(Entry<CharSequence, CharSequence> entry) throws Exception {
     }
 
     /**
+     * Converts HTTP/1.1 trailing headers into HTTP/2 headers
+     */
+    public static Http2Headers toHttp2Headers(LastHttpContent in) throws Exception {
```

Comment:
+1. This code should also be shared with the other method `toHttp2Headers` that also has this code.